test/distributed/test_store.py

```diff
@@ -171,6 +171,7 @@ def test_address_already_in_use(self):
             store1 = dist.TCPStore(addr, port, 1, True)  # noqa: F841
             store2 = dist.TCPStore(addr, port, 1, True)  # noqa: F841
 
+    @retry_on_connect_failures
```

Comment:
Assuming there's no other callsites using find_free_port that don't have this?

Reply:
I did a grep through the repo and this was the only other test that needed this